### PR TITLE
Recruitment1.2.1#55

### LINKED ISSUE #55: As a HR Staff (handling recruitment), I want to add recruitment post(s)
so that i can put up new recruitment post(s) for newly available positions

### PR DESCRIPTION
**Pull request by ryanchen2018**
Resolves #55 issue.

- implementation of addRecruitmentPost command: to add job position to the post.

- create a XML storage for the recruitment post.

- Ui updated

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -120,40 +130,40 @@ private Model initModelManager(Storage storage, UserPrefs userPrefs) {
         }
```
Comment (zhihong8888) [inline src/main/java/seedu/address/MainApp.java]: This file got error. 
suppose to have 3 copies, one recruitment, one schedule, one expenses. 
but now vernon's expenses missing when viewed file*

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -84,7 +100,7 @@ public static Type parseStatus(String status) throws ParseException {
         requireNonNull(status);
         String trimmedStatus = status.trim();
         if (!Type.isValidType(trimmedStatus)) {
-            throw new ParseException(Type.MESSAGE_TYPE_CONSTRAINTS);
+            throw new ParseException(Post.MESSAGE_POST_CONSTRAINTS);
```
Comment (zhihong8888) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: This line not supposed to be changed to Post.MESSAGE_POST_CONSTRAINTS.
revert to Type.MESSAGE_TYPE_CONSTRAINTS)

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 3]
```diff
@@ -35,9 +35,9 @@
 
     // Independent Ui parts residing in this Ui container
```
Comment (zhihong8888) [inline src/main/java/seedu/address/ui/MainWindow.java]: << Vernon's stuff got reverted to old copy>>

File: src/main/resources/view/MainWindow.fxml [hunk 4]
```diff
@@ -80,7 +80,7 @@
                       </TitledPane>
                       <TitledPane animated="True" text="Expenses">
```
Comment (zhihong8888) [inline src/main/resources/view/MainWindow.fxml]: << Vernon's stuff got reverted to old copy>>

File: src/main/java/seedu/address/commons/events/ui/RecruitmentPanelSelectionChangedEvent.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+import seedu.address.model.recruitment.Recruitment;
+
+/**
+ * Represents a selection change in the schedule List Panel
```
Comment (LimYiSheng) [inline src/main/java/seedu/address/commons/events/ui/RecruitmentPanelSelectionChangedEvent.java]: Do update this comment

File: src/main/java/seedu/address/logic/Logic.java [hunk 6]
```diff
@@ -30,6 +31,9 @@
     /** Returns an unmodifiable view of the filtered list of schedule */
     ObservableList<Schedule> getFilteredScheduleList();
 
+    /** Returns an unmodifiable view of the filtered list of schedule */
```
Comment (LimYiSheng) [inline src/main/java/seedu/address/logic/Logic.java]: Do update the comment above

File: src/main/java/seedu/address/logic/commands/AddRecruitmentPostCommand.java [hunk 7]
```diff
@@ -15,19 +17,49 @@
...
             + "IT Manager" + PREFIX_MINIMUM_EXPERIENCE + "3"
             + PREFIX_JOB_DESCRIPTION + "To maintain the network server in company";
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "Post";
+
     public static final String MESSAGE_FAILURE = "Recruitment Posts are failed";
 
+    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+
```
Comment (LimYiSheng) [inline src/main/java/seedu/address/logic/commands/AddRecruitmentPostCommand.java]: Do check your MESSAGE constants, they should be of recruitment posts instead.

**Pull request merged**